src/reservations/reservations.controller.ts

```
@@ -51,6 +50,8 @@
 import { User } from '@/users/entities/user.entity';
 import { FilesService } from '@/files/files.service';
 import { Reservation } from './entities/reservation.entity';
+import { Public } from '@/common/decorator/public.decorator';
```

ESLint

[error] 'Public' is defined but never used.

(@typescript-eslint/no-unused-vars)